mariadb/templates/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.usePersistence }}
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.dataVolume.storageClass | quote }}
```

Comment:
In GKE it works automatically using the dynamic volume provisioning. It currently ignores the storage class but requires it to be there.